tests/queries/0_stateless/00602_throw_if.sh

```diff
@@ -4,9 +4,11 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CURDIR"/../shell_config.sh
 
-default_exception_message="Value passed to 'throwIf' function is non zero"
+default_exception_message="Value passed to 'throwIf' function is non-zero"
 custom_exception_message="Number equals 1000000"
+custom_error_code="42"
```

Comment:
Let's also add tests with some special error codes that can be processed special way (like `UNEXPECTED_PACKET_FROM_CLIENT` in `TCPHandler` or `OK`) and error codes that does not exist (like `123456` or `-1`)

Reply:
Sure, will do.